src/optimum/rbln/diffusers/configurations/models/configuration_transformer_cosmos.py

```diff
@@ -52,6 +52,9 @@ def __init__(
         Raises:
             ValueError: If batch_size is not a positive integer.
         """
+        if kwargs["timeout"] is None:
+            kwargs["timeout"] = 80
```

Comment:
```suggestion
        if kwargs.get("timeout") is None:
```
or use explicit arg of `__init__` func